src/utils/fhirUtils.ts

```diff
@@ -211,3 +212,13 @@ export function createCarePlan(title: string, patient: Patient): CarePlan {
     subject: { reference: `Patient/${patient.id}` }
   };
 }
+
+export function getTNM(mcodeElements: McodeElements): string {
+  let tnm = '';
```

Comment:
clearly this works, but its a lot of `+=` and `??` to get the whitespace right.  Could probably use join here (and avoid a trailing whitespace if there is no `metastatsesCategory`)

I didn't actually test this so I might have a syntax error.

```typescript
return [ mcodeElements.tumorCategory?.split(' ')[0],
              mcodeElements.nodeCategory?.split(' ')[0],
              mcodeElements.metastasesCategory?.split(' ')[0]
           ].join(' ')

Reply:
It would avoid the trailing whitespace but also introduce a double space if `nodeCategory` is undefined. At the end of the day neither are a big deal and I think this suggestion is cleaner so I'll go with it